webrtc.html

```diff
@@ -12409,7 +12409,9 @@ <h3>
                     The {{RTCIceTransport}} has finished gathering, received an
                     indication that there are no more remote candidates,
                     finished checking all candidate pairs, and all pairs have
-                    either failed connectivity checks or have lost consent.
+                    either failed connectivity checks or lost consent, and
+                    either zero local candidates were gathered or the PAC timer
+                    has expired [[RFC8863]].
```

Comment:
Sounds good, we need to figure out whether revealing PAC timer value is a fingerprinting issue or not.
For instance, can it reveal network type? If a fixed value, seems ok

Reply:
[RFC8864 section 4](https://www.rfc-editor.org/rfc/rfc8863.html#section-4) says _"the ICE agent will start its timer once it believes ICE connectivity checks are starting,"_ and _"The RECOMMENDED duration for the PAC timer is equal to the agent's connectivity check transaction timeout, including all retransmissions"_, so this seems network agnostic to me.

Reply:
This looks right to me. The PAC timer is meant to be independent of path type, so I agree there shouldn't be a fingerprinting concern.